.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,36 +4,55 @@ about: Create a report to help us improve
 
 ---
 
-<!--
-PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.
+<!-- Before submitting an issue, please consult our troubleshooting guide (http://ionicframework.com/docs/troubleshooting/) and developer resources (http://ionicframework.com/docs/developer-resources/) -->
 
-ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.
--->
+<!-- Please make sure you are posting an issue pertaining to the Ionic Framework. If you are having an issue with the Ionic Pro services (Ionic View, Ionic Deploy, etc.) please consult the Ionic Pro support portal (http://support.ionicjs.com) -->
 
 # Bug Report
 
-**Ionic Info**
-Run `ionic info` from a terminal/cmd prompt and paste the output below.
+**Ionic version:**
+<!-- (For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1) -->
+<!-- (For Ionic 2.x & 3.x issues, please use https://github.com/ionic-team/ionic-v3) -->
+[x] **4.x**
 
-```
-insert the output from ionic info here
-```
+**I'm submitting a ...**
+
+[x] bug report

```

Comment:
This doesn't need to be here, this template is only ever used by the bug report issue template here: https://github.com/ionic-team/ionic/issues/new?template=bug_report.md

Reply:
Yeah, i let it in because it was in the old template, too ;) I will make the change